src/gui/LayoutMapper.h

```diff
@@ -0,0 +1,150 @@
+/*
+ * Xournal++
+ *
+ * A layout manager - map where( row,column) to which page( document index)
+ *
+ * @author Justin Jones
```

Comment:
Instead writing your name here, add you to the author list in the about dialog, this is probably never read by any user ;-)

Reply:
Maybe I'll earn my way on to the about dialog in the future with a few more commits ;)

Thanks for checking and fixing the numPairsOffset.